portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase

```diff
@@ -65,63 +79,82 @@ definition {
 			entryType = "${customElementType}");
 	}
 
-	@description = "Verify that remote app of type Custom Element can be displayed by portlet"
-	@priority = "5"
-	test CustomElementCanBeDisplayedByPortlet {
```

Comment:
This existing testcase was deleted, please add back...

Reply:
Done.